wire-library/wire-compiler/src/main/java/com/squareup/wire/schema/WireRun.kt

```diff
@@ -148,6 +148,12 @@ data class WireRun(
    */
   val until: String? = null,
 
+  /**
+   * The only version of the version range. Fields with `until` values greater than this, as well as
+   * fields with `since` values less than or equal to this, are retained.
+   */
+  val onlyVersion: String? = null,
```

Comment:
I don't like that this has the suffix of "Version" in the DSL whereas "since" and "until" do not. They should match.